api/controllers/Irc_EntryController.js

```diff
@@ -55,5 +108,53 @@ module.exports = {
       .catch((error) => {
         res.serverError(error.message);
       });
+  },
+
+  process_event: function (request_body) {
+    switch (request_body.operation) {
+    case Event.OPERATION_INTER_SITE_TRANSFER:
+      return this.handleInterSiteTransfer(request_body);
+    case Event.OPERATION_UPDATE:
+      return processEventDetainee(request_body);
+    case Event.OPERATION_CHECK_IN:
+    case Event.OPERATION_CHECK_OUT:
+    case Event.OPERATION_REINSTATEMENT:
+      return processEventDetainee(request_body)
+        .then((detainee) => Event.create(generateStandardEvent(detainee, request_body)));
+    default:
+      throw new ValidationError('Unknown operation');
```

Comment:
<img width="598" alt="screen shot 2016-04-26 at 11 32 45" src="https://cloud.githubusercontent.com/assets/98427/14815327/a1a3896e-0ba2-11e6-8ea2-568bf97c2a79.png">
Why is this a valid ValidationError?